localtypings/react-slick.d.ts

```diff
@@ -0,0 +1,47 @@
+/// <reference path="../typings/globals/react/index.d.ts" />
```

Comment:
Still needed?

Reply:
Yes. For slider. Once @riknoll's carousel PR rolls in, we can remove. This PR depends on react-slick.